src/components/Suggestions/index.css

```diff
@@ -33,6 +33,7 @@
   padding: 0;
   margin: 0;
   scroll-behavior: smooth;
+  margin-top: 7rem;
```

Comment:
you should've checked it for signed in user as well. It's looking bad for logged in user now
![image](https://github.com/narayan954/dummygram/assets/77617189/4f9feaa0-2732-412a-bc4b-05882482a066)

Reply:
Then we can settle it in around 4rem so that with and without login, it will look fine.

Reply:
if that looks fine, it's a green light from my side

Reply:
also, do update your branch with the latest changes